src/components/DataTypes/createEasyType.tsx

```diff
@@ -46,7 +46,7 @@ export function createEasyType<Value> (
       store => store.colorspace[config.colorKey])
     return (
       <InputBase
-        value={value}
+        value={type === 'bigint' ? `${value}` : value}
```

Comment:
just treat everything as `${value}`

Reply:
I think we should detect the type in the `createEasyType` function. That's not what it supposed should do

Reply:
Yeah, type checking needs to be done in the function, not in the expression, but except for `string` and `number`, it seems that there are few types that need to be converted?